h2o-r/tests/testdir_algos/gbm/runit_GBM_offset_poisson.R

```diff
@@ -27,7 +27,7 @@ test <- function(h) {
 	#expect_equal( fit2$train.error[600], hh@model$training_metrics@metrics$MSE,tolerance=1e-6)
 	expect_equal(mean(pr), mean(ph[,1]),tolerance=1e-5 )
 	expect_equal(min(pr), min(ph[,1]) ,tolerance=1e-4)
-	expect_equal(fit2$train.error[length(fit2$train.error)], hh@model$training_metrics@metrics$mean_residual_deviance,tolerance=1e-4) #residual deviance
```

Comment:
why is this commented out?

Reply:
What is the reason here? Does it mean that gbm from R also provides the optimal version of deviance?